test/index-test.js

```diff
@@ -8,7 +8,7 @@ const path = require('path');
 const pathToProcessor = path.join(__dirname, '../index.js');
 
 const config = {
-  processors: [pathToProcessor],
+  processors: [[pathToProcessor, { fileFilterRegex: [] }]],
```

Comment:
Since this should be the default value, I suggest we leave it out of the config unless a test specifically adds it.